buildpack.md

```diff
@@ -95,19 +95,23 @@ Executable: `/bin/build <platform[AR]> <cache[EC]> <launch[EI]>`, Working Dir: `
 | [exit status]                 | Success (0) or failure (1+)
 | `/dev/stdout`                 | Logs (info)
 | `/dev/stderr`                 | Logs (warnings, errors)
-| `<cache>/<layer>/bin/`        | Binaries for subsequent buildpacks
-| `<cache>/<layer>/lib/`        | Libraries for subsequent buildpacks
+| `<cache>/<layer>/bin/`        | Binaries for subsequent buildpacks (added to `PATH` in build phase)
```

Comment:
Note that this content is already provided in the spec [here](
https://github.com/buildpack/spec/blob/master/buildpack.md#provided-by-the-lifecycle).  If you’d ike to duplicate it, you should duplicate all of the special directories.

Reply:
I read that section and didn't join the dots (it's super obscure). I don't really know what you mean by "all the special directories".

Reply:
You've documented how `bin/` goes to `$PATH` and `lib/` goes to `$LD_LIBRARY_PATH`, but you'll need to document `lib/` to `$LIBRARY_PATH`, `/include` to `$CPATH`, and `/pkgconfig` to `$PKG_CONFIG_PATH` as well,

Reply:
I see. That explains the link between this section and the one you linked to. Let's see what it looks like. Pushed as e4c5e06.

Reply:
That commit doesn't seem to be attached to this PR?  I guess you'll need to push to the same branch name?  Or is this because it's based on `master`?